classes/requests/helpers/class-truelayer-helper-verifying.php

```diff
@@ -9,15 +9,15 @@
 	exit;
 }
 
-use TrueLayer\Signing\Exceptions\InvalidSignatureException;
-use TrueLayer\Signing\Verifier;
+use KrokedilTrueLayerDeps\TrueLayer\Signing\Exceptions\InvalidSignatureException;
+use KrokedilTrueLayerDeps\TrueLayer\Signing\Verifier;
 
 /**
- * Heler class for the callback verifying.
+ * Helper class for the callback verifying.
  */
 class Truelayer_Helper_Verifying {
```

Comment:
You also won't need to manually verify webhooks anymore. You can use the [php client to handle webhooks](https://github.com/TrueLayer/truelayer-php?tab=readme-ov-file#webhooks) and signatures will be verified under the hood.